src/main/java/com/hillel/items_exchange/model/Attachment.java

```diff
@@ -0,0 +1,25 @@
+package com.hillel.items_exchange.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Attachment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
Why cascade is here but not into Message?

Reply:
Traces of long attempts to make it work))) Changed. Removed to Message class.